packages/chain-bridge/src/router/BridgeRouter.sol

```diff
@@ -60,15 +66,25 @@ contract BridgeRouter is
     mapping(uint16 chainId => address routerAddress)
         public chainToRouterAddress;
 
+    /// @notice Reference to the cross-chain registry
```

Comment:
let's use the `CrossChainConfigManaged` - same as in onther contract ie https://github.com/OasisDEX/summer-earn-protocol/blob/0440c078102e2bcc789ba9a4e92659fff819afac/packages/core-contracts/src/contracts/FleetProxy.sol#L28